apps/alphabet_mobile/android/app/src/main/java/com/alphabet_mobile/MainActivity.java

```diff
@@ -0,0 +1,32 @@
+package com.alphabet_mobile;
+
+import com.facebook.react.ReactActivity;
+import com.facebook.react.ReactActivityDelegate;
+import com.facebook.react.defaults.DefaultNewArchitectureEntryPoint;
+import com.facebook.react.defaults.DefaultReactActivityDelegate;
+
+public class MainActivity extends ReactActivity {
+
+  /**
+   * Returns the name of the main component registered from JavaScript. This is used to schedule
+   * rendering of the component.
+   */
+  @Override
+  protected String getMainComponentName() {
+    return "alphabet_mobile";
```

Comment:
Decorators everywhere but we can't use an annotation for this. What a shame, haha.